src/pudl/metadata/codes.py

```diff
@@ -1231,4 +1231,256 @@
+                (
+                    "SPRM",
+                    "city_utilities_of_springfield",
+                    "City Utilities Of Springfield, MO",
+                ),
+                (
+                    "SPS",
+                    "southwestern_public_service",
+                    "Southwestern Public Service Co. (Xcel Energy)",
+                ),
+                ("SRP", "salt_river_project", "Salt River Project"),
+                ("SWPP", "southwest_power_pool", "Southwest Power Pool"),
+                ("TAL", "city_of_tallahassee", "City of Tallahassee"),
+                ("TEC", "tampa_electric", "Tampa Electric Company"),
+                ("TEPC", "tuscon_electric_power", "Tucson Electric Power Company"),
+                ("TIDC", "turlock_irrigation_district", "Turlock Irrigation District"),
+                (
+                    "TPWR",
+                    "city_of_tacoma",
+                    "City of Tacoma, Department of Public Utilities, Light Division",
+                ),
+                ("TVA", "tennessee_valley_authority", "Tennessee Valley Authority"),
+                (
+                    "WACM",
+                    "western_area_power_mountain",
+                    "Western Area Power Administration - Rocky Mountain Region",
+                ),
+                (
+                    "WALC",
+                    "western_area_power_southwest",
+                    "Western Area Power Administration - Desert Southwest Region",
+                ),
+                (
+                    "WAUE",
+                    "western_area_power_east",
+                    "Western Area Power Administration - Upper Great Plains East",
+                ),
+                (
+                    "WAUW",
+                    "wester_area_power_west",
+                    "Western Area Power Administration UGP West",
+                ),
+                (
+                    "WFEC",
+                    "western_farmers_electric_coop",
+                    "Western Farmers Electric Cooperative",
+                ),
+                ("WR", "westar_energy", "Westar Energy"),
+                ("WWA", "naturener_wind", "NaturEner Wind Watch, LLC"),
+                (
+                    "YAD",
+                    "alcoa_power_yadkin",
+                    "Alcoa Power Generating, Inc. - Yadkin Division",
+                ),
+                ("GLHB", "GridLiance", "GridLiance (GLHB)"),
+            ],
+        ).convert_dtypes(),
+        "code_fixes": {
+            "NVE": "NEVP",
+            "IS": "ISNE",
+            "PS": "PSCO",
+            "TIC": "TIDC",
+            "TID": "TIDC",
+            "CA": "CISO",
+            # KACY > SWPP
+            # INDN 	Independence Power & Light (Independence,Misso...
+        },
+        "ignored_codes": [
+            "GRID",  # 2022 860m code for plant ID 55328 (code: CSTO for previous years)
```

Comment:
Why not map GRID to CSTO in this case?

Reply:
Yeah i struggled with this one. I'm not sure what the "right" thing is here. It feels hard to know because the 860m does not have the BA names.

Reply:
Yeah fair enough. We're not doing any other time series filling so probably best to not fix this one either yet.